src/main/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModel.java

```diff
@@ -182,7 +182,15 @@ public int getChildCount(Object parent) {
             result = ((DetailNode) parent).getChildren().length;
         }
         else {
-            result = ((DetailAST) parent).getChildCount();
+            if (parseMode == ParseMode.JAVA_WITH_JAVADOC_AND_COMMENTS
+                    && ((AST) parent).getType() == TokenTypes.COMMENT_CONTENT
+                    && JavadocUtils.isJavadocComment(((DetailAST) parent).getParent())) {
+                //Child is javadoc tree
```

Comment:
Please keep anwering to each of points as "done" or ...

Still not clear why 1 is hardcoded why not ((DetailAST) parent).getChildCount ?